apps/calendar/js/main.js

```diff
@@ -34,30 +20,26 @@ require.config({
   }
 });
 
-require([
-  'app',
-  'debug',
-  'next_tick'
-], (app, debug, nextTick) => {
-  debug = debug('main');
-
-  debug('Will wait for window load...');
-  waitForLoad().then(() => {
-    debug('Window loaded!');
-
-    // Restart the calendar when the timezone changes.
-    // We do this on a timer because this event may fire
-    // many times. Refreshing the url of the calendar frequently
-    // can result in crashes so we attempt to do this only after
-    // the user has completed their selection.
-    debug('Will listen for timezone change...');
-    window.addEventListener('moztimechange', () => {
-      debug('Noticed timezone change!');
-      nextTick(() => app.forceRestart(), app._mozTimeRefreshTimeout);
-    });
+// first require.config call is used by r.js optimizer, so we do this second
+// call to list modules that are bundled to avoid duplicate defines
+require.config({
+  paths: {
+    'views/current_time': 'delayed',
+    'views/week': 'delayed',
+    'views/advanced_settings': 'delayed',
+    'views/create_account': 'delayed',
+    'views/day': 'delayed',
+    'views/modify_account': 'delayed',
+    'views/modify_event': 'delayed',
+    'views/settings': 'delayed',
+    'views/view_event': 'delayed'
+  }
+});
 
-    app.init();
-  });
+define(function(require) {
+  'use strict';
+  var app = require('app');
+  app.init();
 });
 
-}());
+require(['main']);
```

Comment:
I think it's a little bit sloppy to have all of this executed in the global space. Can we confine ourselves to a self-executing anonymous function block like we did before?